sbncode/TPCReco/NuVertexChargeTree_module.cc

```diff
@@ -876,7 +835,9 @@ void sbn::NuVertexChargeTree::FillNeutrino(const simb::MCTruth &nu,
 
   art::ServiceHandle<cheat::BackTrackerService> backtracker;
   // TODO: fix -- for now, use a null space-charge service
-  const spacecharge::SpaceCharge *sce = nullptr;
+  const spacecharge::SpaceCharge *sce = lar::providerFrom<spacecharge::SpaceChargeService>();
+  // If we are not correcting for SCE, then blank out the service
+  if (!fCorrectSCE) sce = nullptr;
```

Comment:
I _recommend_:
```suggestion
  // If we are not correcting for SCE, then blank out the service
  const spacecharge::SpaceCharge *sce = fCorrectSCE? lar::providerFrom<spacecharge::SpaceChargeService>(): nullptr;
```
The recommended pattern does not force the presence of `SpaceCharge` service if it is not required.